mindjunkies/forums/views.py

```
@@ -58,43 +62,42 @@ def handle_topic_submission(self, request, course, course_slug):
             return redirect('forum_home', course_slug=course_slug)
-        
+
         # Get the topic
         topic = get_object_or_404(ForumTopic, id=topic_id)
-        
+
         # Create the reply
         reply = ForumReply(
             topic=topic,
             author=request.user,
             content=content
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)